engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -199,10 +201,12 @@ object K8sDeploymentManager {
       (other way to mitigate this would be to generate some hash, but it's a bit more complex...)
     - ensure some level of readability - only id would be hard to match name to scenario
    */
-  private[manager] def objectNameForScenario(processVersion: ProcessVersion, hashInput: Option[String]): String = {
+  private[manager] def objectNameForScenario(processVersion: ProcessVersion, hashInput: Option[String], releaseName: Option[String] = None): String = {
```

Comment:
1. can you remove default value? I could cause that in some places it will be accidentally omitted
2. can it be at the beginning of parameter list like it is rendered at the end?